fjord/suggest/provider_sumosuggest.py

```diff
@@ -0,0 +1,68 @@
+import json
+import logging
+
+import requests
+
+from fjord.suggest import Link, Provider
+
+
+SUMO_SUGGEST_API_URL = 'https://support.mozilla.org/api/2/search/suggest/'
+logger = logging.getLogger('i.dummyprovider')
+
+
+def get_kb_articles(locale, product, text):
+    """Retrieves kb articles using the SUMO search suggest API
+
+    Throws a variety of exceptions all of which the caller should
+    deal with.
+
+    :arg text: The text to search
+    """
+    params = {
+        'q': text,
+        'locale': 'en-US',
+        'product': 'firefox',
+        'max_documents': 3,
+        'max_questions': 0
+    }
+    headers = {
+        'content-type': 'application/json',
+        'accept': 'application/json; indent=4',
```

Comment:
It makes it easier when looking at raw HTTP request/response data, but you're right--I don't need it when not debugging. I'll nix it.